site/docs/4.1/assets/js/src/application.js

```diff
@@ -10,60 +10,96 @@
  * details, see https://creativecommons.org/licenses/by/3.0/.
  */
 
-/* global ClipboardJS: false, anchors: false, Holder: false, bsCustomFileInput: false */
+/* global ClipboardJS: false, anchors: false, Holder: false, bootstrap: false, bsCustomFileInput: false */
 
-(function ($) {
+(function () {
   'use strict'
 
-  $(function () {
+  document.addEventListener('DOMContentLoaded', function () {
     // Tooltip and popover demos
-    $('.tooltip-demo').tooltip({
-      selector: '[data-toggle="tooltip"]',
-      container: 'body'
+    var tooltipDemoList = [].slice.call(document.querySelectorAll('.tooltip-demo'))
```

Comment:
If we don't care about IE, we could just use `forEach` everywhere I think.

Reply:
Edge < 16 does not support forEach on `NodeList` objects, as well Safari < 10.
Is it planned to drop the support for these versions?

Reply:
Nope. I guess we could either polyfill that or add a helper method.

Reply:
```js
var tooltipDemoList = bootstrap.Util.makeArray(document.querySelectorAll('.tooltip-demo'))